package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@sysvale/cuida",
-  "version": "1.3.3",
+  "version": "1.3.4",
```

```suggestion
  "version": "1.3.6",
```
Já foi uma versão 1.3.5